### PR TITLE
chore: add an 'ext_' prefix to external charts config files

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

**Review by dduportal — APPROVED** [review]
🚀 

The errors come from the updatecli 's behavior with SCM/local SCM.

File: config/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+External charts config files are prefixed with "ext_" to be easily noticed.
+
+In these config, we're setting/surcharging and updating the image.tag with udpatecli (See ./updatecli/updatecli.d/docker-images).
```
Comment (timja) [inline config/NOTES.txt:3]: s/udpatecli/updatecli

Comment (lemeurherve) [reply config/NOTES.txt:3]: Grrr... I have to find a tool/trick to avoid this error which bitten me more than once already...

Comment (halkeye) [reply config/NOTES.txt:3]: https://pre-commit.com/index.html with codespell plugin maybe?

Comment (lemeurherve) [reply config/NOTES.txt:3]: I've got the codespell plugin already, reading the pre-commit link, thanks for the suggestion @halkeye :)

**Pull request merged**